Source/Scene/Expression.js

```diff
@@ -43,6 +43,19 @@ define([
         }
     };
 
+    var unaryFunctions = {
+        abs : Math.abs,
+        sqrt : Math.sqrt,
+        cos : Math.cos,
+        sin : Math.sin,
+        tan : Math.tan,
+        acos : Math.acos,
+        asin : Math.asin,
+        atan : Math.atan,
+        radians : CesiumMath.toRadians(),
+        degrees : CesiumMath.toDegrees()
```

Comment:
I redownloaded my repo and even though I'm on my 3d-tiles-unary-ops branch, the 3D tiles style sandcastle file is just gone, with a  lot of others, even though they're showing up  in my repo on github? I don't understand how that's happening